### PR TITLE
Milestones 1, 2 & 3 - Personal Capstone Project #1

### PR DESCRIPTION
**Pull request by Anjacodes**
In this milestone, I:

* Created the "Home" and "About" Mobile Pages Using HTML and CSS
* Used Media Queries to Make the Page Responsive for Desktop
* Implemented the Mobile Menu and Dynamically Generated the "Teachers" Section Using JavaScript. 

Here is the link to the[ Loom video](https://www.loom.com/share/660c9c8ed3eb4408884e5d267a198896), where I present the website.

### CODE REVIEW

File: JavScript/script.js [hunk 1]
```diff
@@ -28,4 +29,63 @@ mobMenuLinks.forEach((link) => {
...
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teacher4 = {
+  image: 'images/chef_square.jpg',
+  alt: 'Female chef portrait',
+  name: 'Amy Meyer',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
```
Comment (Emmyn5600) [inline JavScript/script.js:66]: Let us try to match the provided design. Please refer to the original design to implement it. We are missing some designs for the image.

- [ ] Original
html6

- [ ] Yours

html9

File: JavScript/script.js [hunk 2]
```diff
@@ -28,4 +29,63 @@ mobMenuLinks.forEach((link) => {
...
+  teachers.forEach((teacher) => {
+    const dinContent = document.createDocumentFragment();
+    const newLi = document.createElement('li');
+    // newLi.className = 'works_card';
+
+    newLi.innerHTML = teachersHtmlTemplate(teacher);
+    dinContent.appendChild(newLi);
+    teachersCont.appendChild(dinContent);
+  });
 });
```
Comment (Emmyn5600) [inline JavScript/script.js:91]: Added the footer, however, I would recommend adding `overflow-x: hidden` in your CSS to fix the overflow of the page.
html10

File: README.md [hunk 3]
```diff
@@ -4,9 +4,11 @@
 
 This is an event page to provide information on and promote an upcoming yoga retreat.
 
-<img width="710" alt="Screenshot_mobile_portfolio" src="#">
+<img width="350" alt="Screenshot_mobile" src="images/mobile_screenshot.png">
 
-The website consists of a "home" and an "about" page. 
+<img width="710" alt="Screenshot_desktop" src="images/desktop_screenshot.png">
+
+The website consists of a "home" and an "about" page.
```
Comment (Emmyn5600) [inline README.md:12]: - [ ] Please work on the `MORE` button, if a user clicks on it to show the remaining sections when it comes to the mobile view. 

htmll11

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @Anjacodes 

Good job so far! 👏🏼 

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:

### Changes requested.

- [ ] Check the comment under the review.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: JavScript/script.js (hunk 1 above)
Comment (Anjacodes) [reply JavScript/script.js:66]: Hi Emmyn, 
Thank you for your comment 🙏 . I have implemented the suggested changes as requested: 
Screen Shot 2021-11-17 at 16 31 56

File: JavScript/script.js (hunk 2 above)
Comment (Anjacodes) [reply JavScript/script.js:91]: Thank you Emmyn, 
I have added an overflow-x: hidden to the body element, which successfully removed the side scroll: 

Screen Shot 2021-11-17 at 16 33 47

File: README.md (hunk 3 above)
Comment (Anjacodes) [reply README.md:12]: I have also implemented the "More" button: 
Before pressing the button: 
Screen Shot 2021-11-17 at 16 34 47
After: 
Screen Shot 2021-11-17 at 16 34 54

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Anjacodes, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**